site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -146,6 +146,22 @@
 
   {% if viewer.isGrader() %}
 
+      var notificationSoundState = Cookies.get('notification_sound');
+      window.audible_notifications_enabled = notificationSoundState === 'true';
+
+      var pushNotificationState = Cookies.get('push_notification');
+      var push_notifications_enabled = pushNotificationState === 'true' || localStorage.getItem('office_hour_queue_notifications') === "true";
+
+      if ("Notification" in window && Notification.permission === "granted") {
+          setNotificationState(push_notifications_enabled);
+      }
+      else {
+          setNotificationState(false);
+      }
+
+      document.getElementById('push-notification-switch').checked = push_notifications_enabled;
```

Comment:
You should probably replace the ~4 uses of `document.getElementById()` that you added in this PR with JQuery.